ts/features/bonus/cgn/screens/merchants/CgnMerchantsCategoriesSelectionScreen.tsx

```diff
@@ -60,7 +65,7 @@ const CgnMerchantsCategoriesSelectionScreen = () => {
       return (
         <CgnMerchantCategoryItem
           title={I18n.t("bonus.cgn.merchantDetail.categories.all")}
-          colors={["#475A6D", "#E6E9F2"]}
+          colors={[gradFirstColor, gradSecondColor]}
```

Comment:
maybe we can use the same utility `getGradientColorValues` as we made in the filters collection?